src/databricks/labs/remorph/config.py

```diff
@@ -7,7 +7,7 @@
 @dataclass
 class MorphConfig:
     __file__ = "config.yml"
-    __version__ = 1
+    __version__ = 2
```

Comment:
where's version remapper, if you increment the config file version? addition of the field with a default value is a backwards compatible change, so we don't really require a new config file version.

Reply:
ok will pivot back to original version as 1

Reply:
Done